products/bpmsuite/_common/product.yml

```diff
@@ -4,6 +4,7 @@ product_logo: bpm-suite.png
 has_installer: true
 current_version: 6.2.0.GA
 documentation_minor_version: 6.2
+no_full_subscription: true
```

Comment:
This is a bit confusing. I suggest you invert the flag and name it: 'full_subscription_required'. You'll need to update more .yml files though.